From 0ab1d83ea3a379ed6a003f62e54f8e8cb8905b2f Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <josh@wilsdon.ca>
Date: Wed, 7 Nov 2018 12:01:22 -0800
Subject: [PATCH] TRITON-940 Add metricPorts metadata to binder for cmon-agent
 Reviewed by: Trent Mick <trentm@gmail.com> Reviewed by: Dylan Yep
 <dyep49@gmail.com> Approved by: Trent Mick <trentm@gmail.com> Approved by:
 Dylan Yep <dyep49@gmail.com>

---
 boot/setup.sh | 3 +++
 package.json  | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/boot/setup.sh b/boot/setup.sh
index 85ab7de..75cbbdc 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -156,6 +156,9 @@ else # FLAVOR == "sdc"
     sdc_log_rotation_add zookeeper /var/log/zookeeper/zookeeper.log 1g
     sdc_log_rotation_setup_end
 
+    # Add metadata for cmon-agent discovery
+    mdata-put metricPorts 1053
+
     # All done, run boilerplate end-of-setup
     sdc_setup_complete
 
diff --git a/package.json b/package.json
index 30dbcdf..8f844b0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "binder",
   "description": "SmartDataService DNS Service",
-  "version": "1.2.0",
+  "version": "1.2.1",
   "author": "Joyent (joyent.com)",
   "license": "MPL-2.0",
   "private": true,
-- 
2.21.0

